README.md

```diff
@@ -9,10 +9,47 @@ References:
 
 ## Prerequisites and preparation
 
-- `blankon-keyring` package installed
-- The latest live-build package (https://github.com/BlankOn/live-build/blob/master/build_master.zip) installed
-- Copy `/usr/share/debootstrap/scripts/stretch` to `/usr/share/debootstrap/scripts/verbeek`
-- In `/usr/share/debootstrap/scripts/verbeek`, change the `debian-archive-keyring` string to `blankon-archive-keyring`
+Need live-build version **20191222**
+
+### Install tools:
+```
+sudo apt install debootstrap make git
+git clone https://salsa.debian.org/live-team/live-build.git debian-live-build
+cd debian-live-build
```

Comment:
The latest successful config is based on this [build](https://salsa.debian.org/live-team/live-build/-/pipelines/191043). Since you're removing [this](https://github.com/BlankOn/live-build/blob/master/build_master.zip) as reference, please add a step after this line: checkout to commit `7360d50fa6b7ef4809d28cf873ab7d29dc2a6f97` to assure the config version is always matched in the future.

Reply:
done by last commit